charts/identity-server/values.prod.yaml

```diff
@@ -76,6 +76,7 @@ identity-server:
     AWS__SystemsManager__ParameterStore__DataProtectionPrefix: '/k8s/identity-server/DataProtectionSecret'
     Application__MinCompletionPortThreads: '10'
     AudkenniSettings__Retries: '24'
+    AudkenniSettings__Url: 'https://tqd75.audkenni.is:443/sso/'
```

Comment:
Do we want to modify these values for prod?

Reply:
This PR will not be merged as is to prod. It is just for the feature deployment for now.